.env

```diff
@@ -0,0 +1,7 @@
+QA=1 # Set to 1 to run tests
+ACCESS_TOKEN=eyJhbGciOiJSUzI1NiIsImtpZCI6IjFDQUY4RTY2NzcyRDZEQzAyOEQ2NzI2RkQwMjYxNTgxNTcwRUZDMTkiLCJ0eXAiOiJKV1QiLCJ4NXQiOiJISy1PWm5jdGJjQW8xbkp2MENZVmdWY09fQmsifQ.eyJuYmYiOjE2MDEzODM4MDMsImV4cCI6MTYwMTM4NTYwMywiaXNzIjoiaHR0cHM6Ly9pZGVudGl0eS54ZXJvLmNvbSIsImF1ZCI6Imh0dHBzOi8vaWRlbnRpdHkueGVyby5jb20vcmVzb3VyY2VzIiwiY2xpZW50X2lkIjoiMTlENTg3N0JBN0M3NDY0M0FBQTA1M0RGNTNDRjA2RUIiLCJzdWIiOiIwZGI3Y2UzZDUzYjA1NmYwODFiYjAwOGIwMjc0ZjA2MCIsImF1dGhfdGltZSI6MTYwMTM4Mzc4OSwieGVyb191c2VyaWQiOiJkNzUzNzRlMS1lM2M5LTRlYTAtOTNkYS1iNzRiYWI4NTQ1ZmYiLCJnbG9iYWxfc2Vzc2lvbl9pZCI6ImIzY2FjMzc2ZTAzYjQ2ZDFhMTY0NTU5OWRlYTk5NTNkIiwianRpIjoiN2VkNzI4ZGU4OTU2Mjk2YjQ1M2QyMjk5MTg3ZmQ2YTQiLCJhdXRoZW50aWNhdGlvbl9ldmVudF9pZCI6Ijc1ZTBmOTk3LTkxN2QtNGNkMi1iNTE1LWViMTM3ODBiMmE1OSIsInNjb3BlIjpbInByb2ZpbGUiLCJvcGVuaWQiLCJhY2NvdW50aW5nLnRyYW5zYWN0aW9ucy5yZWFkIl19.OYwUdeufgOiN__WzsoGWgWTmnHgYM-TqcyWkVsmYLDGShxyYfuR913kjo_hedNtH6uhjCXfvk5dHKbsaEKFsrYguhpDQ9bsrkXqGNH7QiBdzdQU3D3xMykMmYQEoL9EKaXqVY08NzW7qdzzGTDrLrppRYEXPVCswuctiEIcS8iMn_qum8rC9lmlaQirVvX10HBPyPDn0HUzJh4GD3W5FHSnk9SCwCurlQ-v-MVdi9_iXu_3h8eidwy_rNQqS0Ou4eRlO3w-xD68ICBByC4GNtYsWZFutzVHrZngXTklbv_-9UGKFGWZxqnOz6ipEvDozGrlpBHmxz-mja_Snpcae5w
+REFRESH_TOKEN=
+CLIENT_ID=C482F81A09BC45DCB6DE2A3B7F5686A9
+CLIENT_SECRET=eWymXZ79Sl5fIOXk94y0IcjJ4snLAP_kPf5jQahjGBZNs2dM
```

Comment:
@amouxkin you should never check this file (`.env`) into github.  These are secrets, so you will need to roll all your secrets, as these are now public.